<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BirthdayListAppearanceConfig</class>
 <widget class="QWidget" name="BirthdayListAppearanceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="3">
    <widget class="QLabel" name="lblTitleComingEvents">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Coming events</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblComingShowDays">
     <property name="text">
      <string>Show events within:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="3">
    <widget class="QLabel" name="lblTitlePastEvents">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Past events</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <spacer name="spacerComingEventsBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblComingHighlightDays">
     <property name="text">
      <string>Highlight events within:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="spinComingHighlightDays">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lblPastShowDays">
     <property name="text">
      <string>Show events within:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QSpinBox" name="spinPastShowDays">
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="lblComingShowDays_2">
     <property name="text">
      <string comment="label: show events within x &lt;days&gt;">days</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="lblComingHighlightDays_2">
     <property name="text">
      <string comment="label: show events within x &lt;days&gt;">days</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QLabel" name="lblPastShowDays_2">
     <property name="text">
      <string comment="label: show events within x &lt;days&gt;">days</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="spinComingShowDays">
     <property name="maximum">
      <number>335</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <layout class="QHBoxLayout" name="hlComingHighlightForeground">
     <item>
      <widget class="QCheckBox" name="chckComingHighlightForeground">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorbtnComingHighlightForeground">
       <property name="color">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="defaultColor">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="1" colspan="2">
    <layout class="QHBoxLayout" name="hlComingHighlightBackground">
     <item>
      <widget class="QCheckBox" name="chckComingHighlightBackground">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorbtnComingHighlightBackground">
       <property name="color">
        <color>
         <red>128</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="defaultColor">
        <color>
         <red>128</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblComingHighlightForeground">
     <property name="text">
      <string>Custom foreground:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblComingHighlightBackground">
     <property name="text">
      <string>Custom background:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3" rowspan="5">
    <spacer name="spacerScreenRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="1" colspan="2">
    <layout class="QHBoxLayout" name="hlPastForeground">
     <item>
      <widget class="QCheckBox" name="chckPastForeground">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorbtnPastForeground">
       <property name="color">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="defaultColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="lblPastForeground">
     <property name="text">
      <string>Custom foreground:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="20" column="1" colspan="2">
    <layout class="QHBoxLayout" name="hlPastBackground">
     <item>
      <widget class="QCheckBox" name="chckPastBackground">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorbtnPastBackground">
       <property name="color">
        <color>
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </property>
       <property name="defaultColor">
        <color>
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="lblPastBackground">
     <property name="text">
      <string>Custom background:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="21" column="0" colspan="3">
    <spacer name="spacerScreenBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinComingShowDays</tabstop>
  <tabstop>spinComingHighlightDays</tabstop>
  <tabstop>chckComingHighlightForeground</tabstop>
  <tabstop>colorbtnComingHighlightForeground</tabstop>
  <tabstop>chckComingHighlightBackground</tabstop>
  <tabstop>colorbtnComingHighlightBackground</tabstop>
  <tabstop>spinPastShowDays</tabstop>
  <tabstop>chckPastForeground</tabstop>
  <tabstop>colorbtnPastForeground</tabstop>
  <tabstop>chckPastBackground</tabstop>
  <tabstop>colorbtnPastBackground</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chckComingHighlightBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorbtnComingHighlightBackground</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckComingHighlightForeground</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorbtnComingHighlightForeground</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckPastBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorbtnPastBackground</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckPastForeground</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorbtnPastForeground</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
